packages/remix/test/utils/getIpAddress.test.ts

```diff
@@ -0,0 +1,37 @@
+import { getClientIPAddress } from '../../src/utils/getIpAddress';
+
+class Headers {
+  private headers: Record<string, string> = {};
+
+  get(key: string): string | null {
+    return this.headers[key] ?? null;
+  }
+
+  set(key: string, value: string): void {
+    this.headers[key] = value;
+  }
+}
+
+describe('getClientIPAddress', () => {
+  it.each([
+    [
+      '2b01:cb19:8350:ed00:d0dd:fa5b:de31:8be5,2b01:cb19:8350:ed00:d0dd:fa5b:de31:8be5, 141.101.69.35',
+      '2b01:cb19:8350:ed00:d0dd:fa5b:de31:8be5',
```

Comment:
This test failed before the test, returning `141.101.69.35` instead of `2b01:cb19:8350:ed00:d0dd:fa5b:de31:8be5`